examples/raw_cloudevent/build.yaml

```diff
@@ -0,0 +1,4 @@
+targets:
```

Comment:
This file isn't needed AT ALL!

It's weird for the JSON example. All about lints. Better to explain over VC

Reply:
ah ok, misunderstood, removing. Yeah would love to understand this better, but it's a saturday -- we can chat next week. Removing file now, thanks!

Reply:
Removed, PTAL when you have a chance, thanks.